recipes/default.rb

```diff
@@ -19,6 +19,8 @@
 # limitations under the License.
 #
 
+include_recipe 'ntp::default' unless node['krb5']['include_ntp'].to_s == 'false'
```

Comment:
btw, why a `to_s` followed by a comparison with 'false'?
Why not just a direct comparison with false?